Add initial content for section "BICEPS Extension Model" (#160)
IHE/sdpi-fhir: SDPi_Supplement/asciidoc/volume3/tf3-ch-8.2.8-biceps-content.adoc

// 8.2.8
[sdpi_offset=8]

> ===== SDC/BICEPS semantic content

Reviewer: @ToddCooper I guess this hierarchy decrement makes sense here, doesn'it it? 8.2.8 is in the comment, it'd missing a fourth digit otherwise.

// 8.2.8.1
===== SDC/BICEPS Content Module

====== BICEPS Extension Model

> IEEE 11073-10207 provides a powerful extension model.

Reviewer: Hmm "powerful" is debatable - it is not less ore more powerful than other extension models (e.g. FHIR)

Reviewer: ```suggestion
IEEE 11073-10207 provides a flexible extension model.
```

This enables manufacturers to add information that does not fit into the defined structure.

> Therefore, nearly every element in the participant and message model contains an extension point (`tns:Extension`).

Reviewer: It is `ext:Extension`.

However, the fundamental rule is: Do not use an extension for anything that can be expressed using the given models.

> Especially, the powerful concept of coded values must be considered before using an extension.

Reviewer: We should elaborate on this one - how does it relate to extension containers? And perhaps also avoid to use the word "must" as it implicates a requirement.

> The boarders of using extension are given by:

Reviewer: ```suggestion
The borders of using extension are given by:
```

> * Clause 8 of <<ieee_11073_10207_2017>>, especially R0109, and

Reviewer: In other clauses, we denoted references by first mentioning the standard followed by the particular clause:

`<<ieee_11073_10207_2017>> Clause 8, especially R0129, [...]`

Reviewer: ```suggestion
* <<ieee_11073_10207_2017>> Clause 8, especially R0109, and
```

> * Clause 6.7 of <<ieee_11073_10700_2022>>

Reviewer: ```suggestion
* <<ieee_11073_10700_2022>> Clause 6.7
```

> There are two kinds of extensions, those, the service consumer must understand to perform system functions safely with the service provider, and those the service consumer does not have to understand.

Reviewer: service consumer -> `<<actor_somds_consumer>>`
service provider -> `<<actor_somds_provider>>`

> This is indicated by the Boolean Attribute `@ext:MustUnderstand`.

Reviewer: boolean attribute

Reviewer: ```suggestion
This is indicated by the Boolean attribute `@ext:MustUnderstand`.
```

> Consequently, a service provider should be careful with defining extension with `@ext:MustUnderstand = true`, as this potentially reduces interoperability.

Reviewer: Unfortuntately, right now we do not have a definition in the glossary for "manufacturer"

> There are extensions that are defined in standards, like the PKP standards. Sometimes, they are used to introduce neu features to the IEEE 11073-10207 models.

Reviewer: There are standardized extensions, like those that are specified in the PKP standards or this profile, which can be considered common knowledge and therefore do not break interoperability between different manufacturers.

Reviewer: ```suggestion
There are standardized extensions, like those that are specified in the PKP standards or this profile, which can be considered common knowledge and therefore do not break interoperability between different manufacturers.
```

// 8.2.8
[sdpi_offset=8]
==== SDC/BICEPS semantic content

// 8.2.8.1
===== SDC/BICEPS Content Module

====== BICEPS Extension Model

IEEE 11073-10207 provides a flexible extension model.
This enables manufacturers to add information that does not fit into the defined structure.
Therefore, nearly every element in the participant and message model contains an extension point (`ext:Extension`).
However, the fundamental rules are: Do not use a proprietary extension if the purpose of the extension can be expressed by (in the specified order)

. the given participant and message models,
. a standardized coded value,
. a standardized extension, or
. a private coded value.

The borders of using extension are given by:

* <<ieee_11073_10207_2017>> Clause 8, especially R0109, and
* <<ieee_11073_10700_2022>> Clause 6.7

> There are two kinds of extensions, those, the <<actor_somds_consumer>> must understand to perform system functions safely with the <<actor_somds_provider>>, and those the <<actor_somds_consumer>> does not have to understand.

Reviewer: ```suggestion
There are two kinds of extensions, those the <<actor_somds_consumer>> must understand to perform system functions safely with the <<actor_somds_provider>>, and those the <<actor_somds_consumer>> does not have to understand.
```

This is indicated by the Boolean attribute `@ext:MustUnderstand`.

> Consequently, a manufacturer of a <<actor_somds_provider>> should be careful with defining extension with `@ext:MustUnderstand = true`, as this potentially reduces interoperability.

Reviewer: ```suggestion
Consequently, a manufacturer of a <<actor_somds_provider>> should be careful defining extensions with `@ext:MustUnderstand = true`, as this potentially reduces interoperability.
```

There are standardized extensions, like those that are specified in the PKP standards or this profile, which can be considered common knowledge and therefore do not break interoperability between different manufacturers.

> Consequently, these extensions can be seen as common knowledge and do not hinder interoperability between different manufacturers.

Reviewer: Remove this sentence since all it expresses is already explained in the previous sentence.

Reviewer 2: Is this extension model in 8.2.8.8?  (the one allocated in the Word draft section 8.2.8)
